catalog/core/catalog-core-solrcommands/src/main/java/org/codice/ddf/commands/solr/SolrCommands.java

```diff
@@ -76,4 +84,35 @@ protected void printErrorMessage(String message) {
         printColor(ERROR_COLOR, message);
     }
 
+    protected void printInfoMessage(String message) {
+        printColor(INFO_COLOR, message);
+    }
+
+    private SolrClient getCloudSolrClient() {
+        String zkHosts = System.getProperty(ZOOKEEPER_HOSTS_PROP);
+        LOGGER.debug("Zookeeper hosts: {}", zkHosts);
+
+        if (zkHosts != null) {
+            SolrClient client = new CloudSolrClient.Builder().withZkHost(zkHosts)
+                    .build();
+            LOGGER.debug("Created solr client: {}", client);
+            return client;
+        } else {
+            printErrorMessage(String.format(
+                    String.format("Could not determine Zookeeper Hosts. Please verify that the system property %s is configured in %s.",
+                    ZOOKEEPER_HOSTS_PROP, Paths.get(System.getProperty("ddf.home"), "etc", "system.properties"))));
+            return null;
+        }
+    }
+
+    void shutdown(SolrClient client) throws IOException {
+        if(client != null) {
+            LOGGER.debug("Closing solr client");
+            client.close();
+        }
+    }
+
+    SolrClient getSolrClient() {
+        return getCloudSolrClient();
```

Comment:
It doesn't look like it.  It looks like calling newClient(String core) creates a core/collection.  We don't want that.
https://github.com/codice/ddf/blob/master/platform/solr/solr-factory/src/main/java/org/codice/solr/factory/SolrClientFactory.java